src/UI/NavigationContainer.elm

```diff
@@ -187,9 +188,17 @@ type Navigator page msg
 type alias NavigatorRecord page msg =
     { container : page -> Container msg
     , menu : Menu.Menu msg
+    , sidebarStyle : SidebarStyle
     }
 
 
+{-| The behavior/appearance that the sidebar will have when enabled
+-}
+type SidebarStyle
```

Comment:
Wouldn't expose the values through functions instead of exposing the union entries be for API purposes?
1 - The type would be opaque;
2 - Adding new values wouldn't break the final app's switch-cases;
3 - Adding new subtypes to those wouldn't break the user instancing functions.

Reply:
generally I'd agree but in this case what `SidebarStyle` is really is a flag with just a teeny tiny more info, it doesnt hold any values and there's no reason to pattern match on it outside the NavigationContainer module